### PR TITLE
Render Relative Content

### PR DESCRIPTION
**Pull request by mcuckson**
Closes airwalk-digital/airview-issues#290

### CODE REVIEW

**Comment by BinaryJim** [comment]
@mcuckson - does this now fix AirWalk-Digital/airview-issues/issues/229 ?

**Comment by mcuckson** [comment]
> @mcuckson - does this now fix [AirWalk-Digital/airview-issues/issues/229](https://github.com/AirWalk-Digital/airview-issues/issues/229) ?

Probaly not, because we still rely on the object url's for previewing images which haven't been uploaded yet.

**Comment by mcuckson** [comment]
Closes airwalk-digital/airview-issues#290

File: packages/airview-cms/src/features/use-get-entry/use-get-entry.js [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
-    ({ data, isSuccess }) => ({
-      isSuccess,
-      entryMetaData: data?.find((entry) => entry.id === entryId),
-    })
-  );
-
-  const entryQuery = useGetEntryQuery(entryMetaData?.sha, {
-    skip: !entryMetaData?.sha,
-  });
+export function useGetEntry({ entryId, path }) {
```
Comment (BinaryJim) [inline packages/airview-cms/src/features/use-get-entry/use-get-entry.js:4]: The `airview-cms` docs will need an update now the arguments have change for this hook

**Review by BinaryJim — APPROVED** [review]

**Pull request merged**